app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ActiveRecord::Base
   validates :email, presence: true, uniqueness: true
   validates :password_digest, presence: true
+
+  def name
```

Comment:
They used this method as an example in the book, and I thought it could be useful to have and a good way to practice testing a method.